forwarder.go

```diff
@@ -148,3 +152,29 @@ func forwarder(c echo.Context) error {
 
 	return nil
 }
+
+
+func parseHost(host string) (string, error) {
```

Comment:
Can we make this more general and set the replacements strings on the cmd line? This would make it usable in more environments. Probably using a regex (regex + replacement) here makes more sense?

Reply:
okay, will do that

Reply:
replaced fixed values with input parameters.